.github/workflows/cd-pipeline.yml

```
@@ -20,7 +20,7 @@ permissions:
 on:
   push:
     branches:
-      - main
+      - 2428-cd-release-notes
```

Just so it's not forgotten.

```suggestion
      - main
```